docs/setup/kubeedge_configure.md

```diff
@@ -0,0 +1,369 @@
+
+    ```
+    To start using your cluster, you need to run the following as a regular user:
+
+    mkdir -p $HOME/.kube
+    sudo cp -i /etc/kubernetes/admin.conf $HOME/.kube/config
+    sudo chown $(id -u):$(id -g) $HOME/.kube/config
+    ```
+
+### Adding the edge nodes (KubeEdge Worker Node) on the Cloud side (KubeEdge Master Node)
+
+We have provided a sample node.json to add a node in kubernetes. Please make sure edge-node is added in kubernetes. Run below steps to add edge-node.
+
++ Copy the `$GOPATH/src/github.com/kubeedge/kubeedge/build/node.json` file and change `metadata.name` to the name of the edge node
+
+    ```shell
+        mkdir -p ~/cmd/yaml
+        cp $GOPATH/src/github.com/kubeedge/kubeedge/build/node.json ~/cmd/yaml
+    ```
+
+    Node.json
+    ```script
+    {
+      "kind": "Node",
+      "apiVersion": "v1",
+      "metadata": {
+        "name": "edge-node",
+        "labels": {
+          "name": "edge-node",
+          "node-role.kubernetes.io/edge": ""
+        }
+      }
+    }
+    ```
+
+#### Modification in node.json
+
+1. metadata.name : This is the name of your edge node device. (Referring the edge node configuration would make it more clear). 
+ 
+    **Note: you need to remember `metadata.name` , because edgecore needs it**.
+
+2. metadata.labels: Labels are the name by which you can remember a set of nodes.
+
+    + Make sure role is set to `edge` for the node. For this a key of the form `"node-role.kubernetes.io/edge"` must be present in `labels` tag of `metadata`.
+    + Please ensure to add the label `node-role.kubernetes.io/edge` to the `build/node.json` file.
+    
+    + If role is not set for the node, the pods, configmaps and secrets created/updated in the cloud cannot be synced with the node they are targeted for.
+
+    + This can be checked by running the below command on Cloud side
+
+        ```
+        kubectl get nodes --show-labels
+        ```
+
+#### Deploy edge node (Run on cloud side)
+
+```shell
+    kubectl apply -f ~/cmd/yaml/node.json
+```
+
+#### Check if the certificates are created (Run on cloud side)
+
+RootCA certificate and a cert/key pair is required to have a setup for KubeEdge. Same cert/key pair can be used in both cloud and edge.
+
+Ideally, when you setup KubeEdge on the cloud side, certificates would have been generated. Check `/etc/kubeedge`. 
+
+If not, perform the following below steps
+
+```shell
+$GOPATH/src/github.com/kubeedge/kubeedge/build/tools/certgen.sh genCertAndKey edge
+```
+
+or 
+
+```shell
+wget -L https://raw.githubusercontent.com/kubeedge/kubeedge/master/build/tools/certgen.sh
+# make script executable
+chmod +x certgen.sh
+bash -x ./certgen.sh genCertAndKey edge
+```
+The cert/key will be generated in the /etc/kubeedge/ca and /etc/kubeedge/certs respectively, so this command should be run with root or users who have access to those directories. 
+We need to copy these files to the corresponding edge side server directory.
```

Comment:
no, you can create new one but they have to use the same certificate authority certificate.

Reply:
but your given solution is correct